webpack.dev.config.babel.js

```diff
@@ -44,7 +44,7 @@ for (const app of appsBuildList) {
 }
 
 export default Object.assign({}, webpackConfig, {
-  devtool: 'cheap-module-eval-source-map',
+  devtool: 'cheap-module-source-map',
```

Comment:
By switching to `cheap-module-source-map`, we don't use `eval` anymore, avoiding LOTS of CSP errors. It is slightly slower apparently, but I did not notice any big impact.